File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/10 - 2 and 3 May 2019/02. Safari.py
budget = float(input())
fuel = float(input())
day = input()

fuel_price = fuel * 2.1

total = fuel_price + 100

if day == "Saturday":
    total -= total * 0.1
elif day == "Sunday":
    total -= total * 0.2

if budget >= total:
    print(f"Safari time! Money left: {abs(budget - total):.2f} lv. ")
else:
    print(f"Not enough money! Money needed: {abs(total - budget):.2f} lv.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/02.  Online Retake Exam - 2 and 3 May 2019/02. Safari.js
function safari(params) {
    let budget = Number(params.shift());
    let price = ((Number(params.shift()) * 2.10) + 100) * {"Saturday": 0.9, "Sunday": 0.8}[params.shift()];

    if (budget >= price) {
        console.log(`Safari time! Money left: ${(budget - price).toFixed(2)} lv.`);
    } else {
        console.log(`Not enough money! Money needed: ${(price - budget).toFixed(2)} lv.`);
    }
}

safari(['1000', '10', 'Sunday']);  // Expected Output:
// Safari time! Money left: 903.20 lv.

safari(['120', '30', 'Saturday']);  // Expected Output:
// Not enough money! Money needed: 26.70 lv.

safari(['105.20', '15', 'Sunday']);  // Expected Output:
// Safari time! Money left: 0.00 lv.
